image_client.py

```diff
@@ -114,6 +113,7 @@ def upload_to_image_server(self, full_path, redacted, collection, original_path=
         r = requests.post(url, files=files, data=data)
         if r.status_code != 200:
             print(f"Image upload aborted: {r.status_code}:{r.text}")
+            add_imagepath_to_txt(local_filename, failure=True)
```

Comment:
if image path gets anything but a 200 code, a --failed to upload flag is added, and it is inserted into the report txt